cogs/stats.py

```diff
@@ -383,11 +388,25 @@ async def callback(self, viewinteract: discord.Interaction):
                                 + len(
                             list(filter(lambda x: x.status == discord.Status.dnd, viewinteract.guild.members)))
                         )
-                    if value == 'members': message = "Участников: %count%"; stat = viewinteract.guild.member_count
-                    if value == 'people': message = "Людей: %count%"; stat = viewinteract.guild.member_count - bot
-                    if value == 'bots': message = "Ботов: %count%"; stat = bot
-                    if value == 'emojis': message = "Эмодзи: %count%"; stat = len(viewinteract.guild.emojis)
-                    if value == 'voice': message = "В войсах: %count%"; stat = voices
+                    match value:
+                        case 'members':
+                            message = "Участников: %count%"
+                            stat = viewinteract.guild.member_count
+                    
+                        case 'people': 
+                            message = "Людей: %count%"
+                            stat = viewinteract.guild.member_count or 0 - bot
```

Comment:
```suggestion
                            stat = (viewinteract.guild.member_count or 0) - bot
```
Я уже писал, почему так